typescript/infra/src/config/gas-oracle.ts

```diff
@@ -257,6 +253,21 @@ export function getAllStorageGasOracleConfigs(
   getOverhead: (local: ChainName, remote: ChainName) => number,
   applyMinUsdCost: boolean = true,
 ): AllStorageGasOracleConfigs {
+  // Ensure all chains have token prices and gas prices by adding stub values
+  chainNames.forEach((chain) => {
```

Comment:
should we remove the chicken and egg handling in the SDK then? or is this still useful for the CLi
https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/mo/balance-threshold-automation/typescript/sdk/src/gas/utils.ts#L119